atlasdb-client/src/main/java/com/palantir/atlasdb/AtlasDbMetricNames.java

```diff
@@ -104,6 +104,7 @@ private CellFilterMetrics() {
     public static final String LW_CACHE_GET_ROWS_HITS = "lockWatchCacheGetRowsHits";
     public static final String LW_CACHE_GET_ROWS_CELLS_LOADED = "lockWatchGetRowsCellsLoaded";
     public static final String LW_CACHE_GET_ROWS_ROWS_LOADED = "lockWatchCacheGetRowsRowsLoaded";
+    public static final String LW_CACHE_ALL_STATE_CLEARED = "allCacheStateCleared";
```

Comment:
nit: can we rename to maybe `lockWatchValueCacheStateCleared`, so that we can differentiate between the event and value caches?